### ISSUE TITLE
Known bugs

### ISSUE THREAD

**Issue by mem48** [SDCA-tool/sdca-package#2]
List of interventions that fail:

- [ ] New rural cycle track 
No components
https://dev.carbon.place/api/v1/locations.json?geojson=%7B%22_timestamp%22%3A1645089991336%2C%22type%22%3A%22FeatureCollection%22%2C%22features%22%3A%5B%7B%22_interventionTypeIndex%22%3A24%2C%22type%22%3A%22Feature%22%2C%22properties%22%3A%7B%22infrastructure_type%22%3A%22transport%22%2C%22mode_class%22%3A%22Active%20travel%22%2C%22mode%22%3A%22Bicycle%22%2C%22intervention_class%22%3A%22New%20construction%22%2C%22intervention_name%22%3A%22New%20rural%20cycle%20track%22%2C%22intervention%22%3A%22cycle_track_rural%22%7D%2C%22geometry%22%3A%7B%22type%22%3A%22LineString%22%2C%22coordinates%22%3A%5B%5B-2.579337%2C51.523184%5D%2C%5B-2.607401%2C51.416772%5D%5D%7D%7D%5D%7D

**Comment by mem48** [comment]
Upgraded cycle schemes never pay back - no data in the demand model

**Comment by mem48** [comment]
- [ ] New high speed line in a cutting 

No materials emissions - bug in how non-granular data is handled

**Comment by mvl22** [comment]
In that second example, the package ought to handle that situation with an error: “We currently don’t have data for this type of intervention yet.”. The package ought to have that kind of generic message at various points.

**Comment by mem48** [comment]
True, I'll get to error handling, but most of these bugs are in the process of being fixed. IT just helps me to have a todo list.
